src/Tizen.NUI/src/public/BaseComponents/TextConstants.cs

```diff
@@ -546,6 +547,74 @@ public struct TextFit : IEquatable<TextFit>
         public override int GetHashCode() => (Enable, MinSize, MaxSize, StepSize, FontSizeType, FontSize).GetHashCode();
     }
 
+    /// <summary>
+    /// A struct to pass data of TextFitArray's OptionList.
+    /// </summary>
+    /// <remarks>
+    /// The TextFitArrayOption struct is used as an item to TextFitArray's OptionList. <br />
+    /// See <see cref="Tizen.NUI.Text.TextFitArray"/>.
+    /// </remarks>
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1815: Override equals and operator equals on value types")]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public struct TextFitArrayOption
+    {
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public TextFitArrayOption(float pointSize)
+        {
+            PointSize = pointSize;
+            MinLineSize = null;
+        }
+
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public TextFitArrayOption(float pointSize, float? minLineSize)
+        {
+            PointSize = pointSize;
+            MinLineSize = minLineSize;
+        }
+
+        /// <summary>
+        /// Point size for text fit array.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public float PointSize { get; set; }
+
+        /// <summary>
+        /// Min line size for text fit array.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public float? MinLineSize { get; set; }
+    }
+
+    /// <summary>
+    /// A struct to pass data of SetTextFitArray and GetTextFitArray methods.
+    /// </summary>
+    /// <remarks>
+    /// The TextFitArray struct is used as an argument to SetTextFitArray and GetTextFitArray methods. <br />
+    /// See <see cref="Tizen.NUI.BaseComponents.TextLabel.SetTextFitArray"/> and <see cref="Tizen.NUI.BaseComponents.TextLabel.GetTextFitArray"/>.
+    /// </remarks>
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1815: Override equals and operator equals on value types")]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public struct TextFitArray
+    {
+        /// <summary>
+        /// True to enable the text fit array or false to disable (the default value is false).
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool Enable { get; set; }
```

Comment:
If the default value of `Enable` is false, then you can set like this :
```suggestion
        public bool Enable { get; set; } = false;
```
It's an another way to tell the default value. :)

Reply:
I tried it and got this error, 
`error CS8400: Feature 'struct field initializers' is not available in C# 8.0. Please use language version 10.0 or greater.`

Do you know how to solve thiS?

Reply:
Oh.. I think we need to update the language version or remove it.
But, I'm not sure the reason we added it before.

```
// TizenFX/src/Tizen.NUI/Tizen.NUI.csproj
<LangVersion>8.0</LangVersion>
```

@dongsug-song (동석님) Can we update the version like this?
`<LangVersion>10.0</LangVersion>`
or
`<LangVersion>latest</LangVersion>`

Reply:
good suggestion!!!
as I know net6 supports LangVersion 10, so it seems to be OK, (tizen8.0 is on net6)
and I think it would be better to write the 10.0 explicitly (not latest) for readability.

Reply:
Okay! I'll update the version in another PR.
Thank you for checking my question!